tests/test_files/tem/sim_config.yaml

```diff
@@ -19,16 +21,16 @@
     electron_dose_std_e_per_nm2: 0   # standard deviation of dose per image. Range : [0,1]. Default : 0.
 
   optics_parameters:
-    magnification: 81000        # the magnification of the simulated microscope. Range : > 1.
-    spherical_aberration_mm: 2.7     # the spherical aberration of the optical system in mm.
-    chromatic_aberration_mm: 2.7     # the chromatic aberration of the optical system in mm.
-    aperture_diameter_um: 50         # diameter of aperture in back focal plane in μm.
-    focal_length_mm: 3.5             # the focal length of primary lens in mm.
-    aperture_angle_mrad: 0.1         # the aperture angle of beam furnished by condenser lens in mrad.
-    defocus_um: 1.0                  # [μm]
-    defocus_syst_error_um: 0.0       # [μm]
-    defocus_nonsyst_error_um: 0.0    # [μm]
-    optics_defocusout: None     # file to write defocus values
+    magnification: 81000             #
```

Comment:
Why removing the descriptions of the parameters?

Reply:
Thank you for catching this. These seem to have been accidentally removed during a merge conflict. The descriptions have been added back.